src/events/http.js

```diff
@@ -279,7 +279,11 @@ function httpWrapper(wrappedFunction) {
                 this, buildParams(url, options, patchedCallback)
             );
 
-            if (options && options.epsagonSkipResponseData) {
+            if (
+                options &&
+                options.epsagonSkipResponseData &&
```

Comment:
shouldn't that be:

`if (options && options.epsagonSkipResponseData || config.getConfig().disableHttpResponseBodyCapture)` ?

Reply:
We want to enable this tracing for fetch-h2 only if all is true:
1. `options` exists.
2. `options.epsagonSkipResponseData` is set to true
3. Nobody disabled the response capture: `!config.getConfig().disableHttpResponseBodyCapture`

This handling is specific to fetch-h2. Later in the module, there's an handling for the rest of the libs.

Reply:
but this is exactly what isn't working. the shimmer doesn't detect fetch-h2, and doesn't set the `options.epsagonSkipResponseData`. so we can force it with `disableHttpResponseBodyCapture`..maybe I don't understand the flags....

Reply:
anyways...it seems to work now :-)